squash_bot/match_tracker/formatters.py

```diff
@@ -39,27 +39,15 @@ def format_matches(cls, matches: dataclasses.Matches) -> str:
                 player_results[match.loser].get("losses", 0) + 1
             )
 
-            point_difference = match.winner_score - match.loser_score
-            player_results[match.winner]["point_difference"] = (
-                player_results[match.winner].get("point_difference", 0) + point_difference
-            )
-            player_results[match.loser]["point_difference"] = (
-                player_results[match.loser].get("point_difference", 0) - point_difference
-            )
-
         player_rows = []
         for player, results in player_results.items():
             wins = results.get("wins", 0)
             losses = results.get("losses", 0)
 
-            point_difference = results.get("point_difference", 0)
-            point_difference_sign = "+" if point_difference > 0 else ""
-            point_difference_str = f"{point_difference_sign}{point_difference}"
-
-            player_rows.append([player.name, wins, losses, point_difference_str])
+            player_rows.append([player.name, wins, losses])
 
         inner_message = tabulate.tabulate(
-            player_rows, ["Player", "Wins", "Losses", "PD"], tablefmt="rounded_grid"
+            player_rows, ["Player", "Wins", "Losses"], tablefmt="rounded_grid"
```

Comment:
The order of the columns are quite loosely bound - it would be easy to change one and not the other.
Not too important of course, but in future it might be worth thinking up some way to make sure the headers of the columns align with the columns set in the `player_rows`.
Potentially at the top of the for loop you could do something like
```
player_stats = {
  "Player": player.name,
  "Wins": results.get("wins", 0),
  "Losses": results.get("losses", 0),
}

...
player_rows.append([val for val in player_stats.values])

inner_message = tabulate.tabulate(
  player_rows, player_stats.keys, ...
)

```